tooling/ast_fuzzer/src/program/mod.rs

```diff
@@ -312,15 +336,15 @@ impl Context {
 
     /// Generate random function body.
     fn gen_function(&mut self, u: &mut Unstructured, id: FuncId) -> arbitrary::Result<()> {
-        self.gen_function_with_body(u, id, |u, fctx| fctx.gen_body(u))
+        self.gen_function_with_body(Some(u), id, |u, fctx| fctx.gen_body(u.unwrap()))
     }
 
     /// Generate function with a specified body generator.
     fn gen_function_with_body(
         &mut self,
-        u: &mut Unstructured,
+        u: Option<&mut Unstructured>,
         id: FuncId,
-        f: impl Fn(&mut Unstructured, FunctionContext) -> arbitrary::Result<Expression>,
+        f: impl Fn(Option<&mut Unstructured>, FunctionContext) -> arbitrary::Result<Expression>,
```

Comment:
Oh, I didn't think an optional unstructured, just something like passing `&mut Unstructured::new(&[])` in `program_wrap_expression`.